parts/linux/cloud-init/artifacts/components.json

```diff
@@ -822,6 +822,35 @@
           }
         }
       }
+    },
+    {
+      "name": "containerd-wasm-shims",
+      "downloadLocation": "/usr/local/bin",
+      "downloadURIs": {
+        "default": {
+          "current": {
+            "versionsV2": [
+              {
+                "renovateTag": "<DO_NOT_UPDATE>",
+                "latestVersion": "0.3.0"
+              },
+              {
+                "renovateTag": "<DO_NOT_UPDATE>",
+                "latestVersion": "0.5.1"
+              },
+              {
+                "renovateTag": "<DO_NOT_UPDATE>",
+                "latestVersion": "0.8.0"
+              },
+              {
+                "renovateTag": "<DO_NOT_UPDATE>",
+                "latestVersion": "0.15.1"
+              }
+            ],
+            "downloadURL": "https://acs-mirror.azureedge.net/containerd-wasm-shims/v${version}/linux/${CPU_ARCH}"
```

Comment:
For shim v0.15.1, the base URL is `spinkube` instead of `containerd-wasm-shims`

Reply:
do you think it would make sense to have a second binary in components.json for containerd-wasm-shims